run.cmd

```diff
@@ -1,19 +1,24 @@
 @if "%_echo%" neq "on" echo off
 setlocal
 
-if defined VisualStudioVersion goto :Run
+if defined VisualStudioVersion (
+    if not defined __VSVersion echo %__MsgPrefix%Detected Visual Studio %VisualStudioVersion% developer command ^prompt environment
+    goto :Run
+)
 
+echo %__MsgPrefix%Searching ^for Visual Studio 2017 or 2015 installation
```

Comment:
@weshaggard  Should I remove this comment as well?

Reply:
Yeah I don't think it adds much value. What would actually add more value is a message that says which version of VsDevCmd it is running. I've wanted to know that on a number of occasions.

Reply:
You mean 4 number version of VS?

Reply:
I think just the full path to VSDevCmd would be enough to point at which VS was installed but if there is also a version that would be good as well. I know the VS2017 prompt prints out a version already which is useful as well but that only works in VS2017.

Reply:
I will try to get something reasonable printed